src/ctia/events/producers/es/mapping.clj

```diff
@@ -17,7 +17,7 @@
     :properties
     {:owner string
      :timestamp ts
-     :model {:type "object"}
+     :object {:type "object"}
```

Comment:
My preference would be to call it entity instead of object, because it is more specific.  On the other hand, I don't think it matters much.